rero_ils/modules/loans/cli.py

```diff
@@ -185,8 +185,8 @@ def get_loanable_items():
     """."""
     # TODO: to be replace by circulation policies
     exclude_item_type_pids = [
-        ItemType.get_pid_by_name('no-checkout'),
-        ItemType.get_pid_by_name('on-site')
+        ItemType.get_pid_by_name('No checkout'),
```

Comment:
We need to move this logic to circ policies ASAP. if a librarian changes the name of the item type, this will not work

Reply:
Yes, you're right. I think that's why there's a `TODO` on line 186. It was already there.